06_modules_lab/03.py

```diff
@@ -1,11 +1,24 @@
-""" Write a program that searches current working directory
-for files larger than 1MB. Every time you find such a file print
-its name to the user.
-
-- When the program finds a large file. It should ask the user
-  a message asking if she wants to delete it, and delete the
-  file if requested
-
-- Take threshold and path as command line arguments
-"""
-
+import sys
+import os
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("Directory")
+parser.add_argument("FileSize")
+args = parser.parse_args()
+
+# (_,dir,fileSize)= sys.argv
+
+for root,dirs,files in os.walk(args.Directory):
+    for name in files:
+        fullFilePath = "{}\{}".format(root,name)

```

Comment:
There's `os.path.join` function which joins directory and name, taking care of the differences between unix and windows (unix path separator is /)